frontend/src/app/About/index.tsx

```diff
@@ -1,3 +1,194 @@
+
+import { Leaf, LightBulbOn, EmojiTalkingHappy } from 'iconoir-react';
+import Airtable from 'airtable';
+import { useState, useEffect } from 'react';
+import Contributors from './Contributors';
+
+
+//import CurrentContributors from '../components/About/CurrentContributors';
+//import PastContributors from '../components/About/PastContributors';
+
+import AboutCarousel from './AboutCarousel';
+
+import styles from "./About.module.scss"
+
+export interface contributorStructure {
+  name: string;
+  gradYr: number;
+  role: string;
+  img: {
+    seriousBase64: string,
+    sillyBase64: string | null,
+  };
+  websiteURL: string;
+  isAlumni: boolean
+};
+
 export default function About() {
-  return <div style={{ flexGrow: 1 }}></div>;
-}
+
+
+
+
+  const [currContributors, setCurrContributors] = useState<contributorStructure[]>([]);
+  const [alumniContributors, setAlumniContributors] = useState<contributorStructure[]>([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      await airtableDB();
+    }
+    fetchData()
+  }, [])
+
+
+  async function convertURLToBase64(url: string) {
+    try {
+      // Fetch the image
+      const response = await fetch(url);
+      if (!response.ok) throw new Error('Network response was not ok.');
+
+      // Convert the image to a Blob
+      const blob = await response.blob();
+
+      // Create a FileReader to convert the Blob into a Base64 string
+      const reader = new FileReader();
+
+      // Return a promise that resolves with the Base64 string
+      return new Promise((resolve, reject) => {
+        reader.onloadend = () => {
+          // Extract the Base64 string (remove the Data URL part)
+          const result = reader.result as string;
+          var base64String
+          if (result) {
+            base64String = result.replace(/^data:.+;base64,/, '');
+          }
+          else {
+            base64String = ''
+          }
+          resolve(base64String);
+        };
+
+        reader.onerror = error => {
+          reject(error);
+        };
+
+        // Read the Blob as a Data URL
+        reader.readAsDataURL(blob);
+      });
+    } catch (error) {
+      console.error('Error converting image to Base64:', error);
+      throw error;
+    }
+  }
+
+  //get data from airtable
+  function airtableDB() {
```

Comment:
It might be better practice to separate these functions from your component and into a `.ts` file. I would try to keep as much logic outside of the React component as possible (to a reasonable extent).